clients/cli/src/main/java/org/apache/gravitino/cli/TestableCommandLine.java

```diff
@@ -21,118 +21,8 @@
-import org.apache.gravitino.cli.commands.RemoveRoleFromGroup;
-import org.apache.gravitino.cli.commands.RemoveRoleFromUser;
-import org.apache.gravitino.cli.commands.RemoveSchemaProperty;
-import org.apache.gravitino.cli.commands.RemoveTableProperty;
-import org.apache.gravitino.cli.commands.RemoveTagProperty;
-import org.apache.gravitino.cli.commands.RemoveTopicProperty;
-import org.apache.gravitino.cli.commands.RevokePrivilegesFromRole;
-import org.apache.gravitino.cli.commands.RoleAudit;
-import org.apache.gravitino.cli.commands.RoleDetails;
-import org.apache.gravitino.cli.commands.SchemaAudit;
-import org.apache.gravitino.cli.commands.SchemaDetails;
-import org.apache.gravitino.cli.commands.ServerVersion;
-import org.apache.gravitino.cli.commands.SetCatalogProperty;
-import org.apache.gravitino.cli.commands.SetFilesetProperty;
-import org.apache.gravitino.cli.commands.SetMetalakeProperty;
-import org.apache.gravitino.cli.commands.SetOwner;
-import org.apache.gravitino.cli.commands.SetSchemaProperty;
-import org.apache.gravitino.cli.commands.SetTableProperty;
-import org.apache.gravitino.cli.commands.SetTagProperty;
-import org.apache.gravitino.cli.commands.SetTopicProperty;
-import org.apache.gravitino.cli.commands.TableAudit;
-import org.apache.gravitino.cli.commands.TableDetails;
-import org.apache.gravitino.cli.commands.TableDistribution;
-import org.apache.gravitino.cli.commands.TablePartition;
-import org.apache.gravitino.cli.commands.TableSortOrder;
-import org.apache.gravitino.cli.commands.TagDetails;
-import org.apache.gravitino.cli.commands.TagEntity;
-import org.apache.gravitino.cli.commands.TopicDetails;
-import org.apache.gravitino.cli.commands.UntagEntity;
-import org.apache.gravitino.cli.commands.UpdateCatalogComment;
-import org.apache.gravitino.cli.commands.UpdateCatalogName;
-import org.apache.gravitino.cli.commands.UpdateColumnAutoIncrement;
-import org.apache.gravitino.cli.commands.UpdateColumnComment;
-import org.apache.gravitino.cli.commands.UpdateColumnDatatype;
-import org.apache.gravitino.cli.commands.UpdateColumnDefault;
-import org.apache.gravitino.cli.commands.UpdateColumnName;
-import org.apache.gravitino.cli.commands.UpdateColumnNullability;
-import org.apache.gravitino.cli.commands.UpdateColumnPosition;
-import org.apache.gravitino.cli.commands.UpdateFilesetComment;
-import org.apache.gravitino.cli.commands.UpdateFilesetName;
-import org.apache.gravitino.cli.commands.UpdateMetalakeComment;
-import org.apache.gravitino.cli.commands.UpdateMetalakeName;
-import org.apache.gravitino.cli.commands.UpdateTableComment;
-import org.apache.gravitino.cli.commands.UpdateTableName;
-import org.apache.gravitino.cli.commands.UpdateTagComment;
-import org.apache.gravitino.cli.commands.UpdateTagName;
-import org.apache.gravitino.cli.commands.UpdateTopicComment;
-import org.apache.gravitino.cli.commands.UserAudit;
-import org.apache.gravitino.cli.commands.UserDetails;
+
+import org.apache.gravitino.cli.commands.*;
```

Comment:
We use Spotless, and this is likely to break the build, please don't use wildcards.

Reply:
@justinmclean Can you elaborate why this revised import is not an improvement?

Reply:
We don't use wildcard imports, and that is set up in a rule in Spotless. You'll need to ask @jerryshao why.

Reply:
I just found the customized rule here:
https://github.com/apache/gravitino/blob/main/build.gradle.kts#L140-L144

Reply:
Please check this (https://errorprone.info/bugpattern/WildcardImport#:~:text=Wildcard%20imports%20are%20forbidden%20by,for%20programmers%20and%20for%20tools.).

Reply:
Makes sense, seems like that got automated somehow. Fixed.

Reply:
Thanks for the pointer, @jerryshao. I do agree that for each and every rule in a lint tool, there is a very clear and solid reason. Java is no exception.
However, I don't think that rule applies universally to all cases.
On the contrary, my gut feeling is that the specific scenario this rule is protecting is a trivial one. It may not even apply to Gravatino.
The fact that this rule has to be EXPLICITLY ENABLED in the gradle script may be a hint for us. It is not something mandatory for most projects.

The stackoverflow discussion has some valid points as well, despite the disputes aroused. `import java.awt.*` is cumbersome and stupid, because it may introduce a lot of name conflicts IF you are overriding a lot of the existing widgets. However, import all exception classes under my own package `very.specific.package.to.my.project.exceptions.*` should be a good practice.
We know what we are doing. We create rules and abide by them when it improves code quality and/or productivity. We ignore/abandon rules when they are counterproductive and/or harmful to the code base.